views/index.html

```diff
@@ -12,18 +12,76 @@
 
     <!-- VENDOR SCRIPTS -->
     <script src="//ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js"></script>
+    <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
 
     <!-- APPLICATION SCRIPTS -->
     <script src="/scripts/app.js"></script>
   </head>
   <body>
-    <div class="container">
-      <div class="row">
-        <div class="col-md-8 col-md-offset-2">
-          <h1>Under Construction</h1>
-          <h4><a href="/api">Read My API Documentation</a></h4>
-        </div>
+    <body>
+    <div class="jumbotron">
+      <div class="container">
+        <h1>Skatespot.com</h1>
+        <p>Find a Skatespot!</p>
       </div>
     </div>
+
+    <!-- Begin form Section -->
+    <section id="skatepark-form" class="container">
+      <!-- a row can be any size in height, grows   bootstrap -->
+      <div class="row"> <!-- a row just sets up the containters -->
+      <!-- offset is spacing, offest by one leaves the 1st container empty
+      and starts the div one over from the left, since there are 10
+      it centers it, leaving one column empty on both sides   -->
+      <div class="col-lg-10 col-lg-offset-1">
```

Comment:
I'm concerned that you're using col-lg in some places, and col-md in others--you generally want to use the smallest possible size for everything, so make all of them col-md or col-sm.